.github/workflows/ci.yml

```diff
@@ -378,25 +369,11 @@ jobs:
       BUILD_DIR: ${{ needs.build_all.outputs.build-dir }}
       BUILD_DIR_ARCHIVE: ${{ needs.build_all.outputs.build-dir-archive }}
       BUILD_DIR_GCS_ARTIFACT: ${{ needs.build_all.outputs.build-dir-gcs-artifact }}
-      TF_BINARIES_DIR: ${{ needs.build_tf_integrations.outputs.binaries-dir }}
-      TF_BINARIES_ARCHIVE: ${{ needs.build_tf_integrations.outputs.binaries-archive }}
-      TF_BINARIES_GCS_ARTIFACT: ${{ needs.build_tf_integrations.outputs.binaries-gcs-artifact }}
     steps:
       - name: "Checking out repository"
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
         with:
           submodules: true
-      - name: "Downloading TF binaries archive"
-        run: gcloud storage cp "${TF_BINARIES_GCS_ARTIFACT}" "${TF_BINARIES_ARCHIVE}"
-      - name: "Extracting TF binaries archive"
-        run: tar -xvf "${TF_BINARIES_ARCHIVE}"
-      - name: "Symlinking TF binaries"
-        run: |
-          ./integrations/tensorflow/symlink_binaries.sh "${TF_BINARIES_DIR}"
-      - name: "Downloading build dir archive"
-        run: gcloud storage cp "${BUILD_DIR_GCS_ARTIFACT}" "${BUILD_DIR_ARCHIVE}"
-      - name: "Extracting build dir archive"
-        run: tar -xf "${BUILD_DIR_ARCHIVE}"
```

Comment:
Something in here was load-bearing... might need to pull the build or install directory from a previous job still

https://github.com/openxla/iree/actions/runs/4862317331/jobs/8668734663?pr=13361#step:4:250
```
$ ":" "RUN: at line 2"
$ "/usr/bin/python3" "-m" "iree_tf_tests.layers.layers_test" "--target_backends=iree_llvmcpu" "--dynamic_dims=false" "--training=false" "--test_default_kwargs_only=true" "--layer=SpatialDropout3D" "--artifacts_dir=/tmp/lit/iree_tf_tests/layers/Output/llvmcpu__SpatialDropout3D.run.tmp"
# command stderr:
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 192, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/work/integrations/tensorflow/test/python/iree_tf_tests/layers/layers_test.py", line 17, in <module>
    from iree.tf.support import tf_test_utils
  File "/work/integrations/tensorflow/python_projects/iree_tf/iree/tf/support/tf_test_utils.py", line 28, in <module>
    from iree.tf.support import module_utils
  File "/work/integrations/tensorflow/python_projects/iree_tf/iree/tf/support/module_utils.py", line 15, in <module>
    import iree.compiler.tf
ModuleNotFoundError: No module named 'iree.compiler'
```

Reply:
Yeah you still need the build directory. Otherwise you don't have IREE at all :-)